packages-web/badge-button/README.md

```diff
@@ -21,10 +20,9 @@ Mendix 7.13.1
 
 ## Usage
 
-Place the widget in the context of an object that has a value attribute.
+Just drag and drop inside or outside a context
```

Comment:
Replace with: Just drag and drop the badge button inside or outside a context.

Reply:
Done